admin/themes/default/template/controllers/modules/js.tpl

```diff
@@ -34,7 +34,7 @@
 	var currentIndexWithToken = '{$currentIndex|escape:'html':'UTF-8'}&token={$token|escape:'html':'UTF-8'}';
 	var dirNameCurrentIndex = '{$dirNameCurrentIndex}';
 	var ajaxCurrentIndex = '{$ajaxCurrentIndex}';
-	var installed_modules = {if isset($installed_modules) && count($installed_modules)}{$installed_modules}{else}false{/if};
+	var installed_modules = {if isset($installed_modules) && is_array($installed_modules) && count($installed_modules) > 0}{$installed_modules}{else}false{/if};
```

Comment:
No need to compare `count($installed_modules) > 0`.

Reply:
@wdammak 
waiting for an update.

Reply:
@shreesh-webkul done.

Reply:
@wdammak 
There has been a misunderstanding. 
Actually, we need to check `count($installed_modules)`, the changes required were only to remove its comparison with 0.
It should be
`{if isset($installed_modules) && is_array($installed_modules) && count($installed_modules)}`

Reply:
@shreesh-webkul done.

Reply:
@wdammak prefect!.